# Should be fixed in upstream trunk, then next upstream
# release shouldn't need this. For more info:
# https://bugs.launchpad.net/ubuntu/+source/gphpedit/+bug/46487 
--- gphpedit-0.9.91.orig/src/find_replace.c
+++ gphpedit-0.9.91/src/find_replace.c
@@ -478,7 +478,10 @@
 
 	replace_all_dialog = gnome_ok_dialog(message->str);
 	gnome_dialog_run_and_close(GNOME_DIALOG(replace_all_dialog));
-	g_free(replace_all_dialog);
+	// Comment the line bellow as suggested at 
+	// http://www.gphpedit.org/bugs/bug_view.php?id=132 to avoid from
+	// a crash after searching & replacing.
+	//g_free(replace_all_dialog);
 	
 	gtk_scintilla_goto_pos(GTK_SCINTILLA(main_window.current_editor->scintilla), start_pos);
 }
